document/core/exec/relaxed.rst

```diff
@@ -1,8 +1,30 @@
+.. todo:: add index entries
+
+.. index:: ! relaxed memory, memory
+.. _relaxed:
+
 Relaxed Memory Model
 --------------------
 
+.. todo:: intro, cite [#cite-oopsla2019]_
 .. todo:: link this section with operational semantics
 
+
+.. _relaxed-trace:
+
+Traces
+~~~~~~
+
+.. todo:: define, explain
+
+
+.. _relaxed-valid:
+
+Validity
+~~~~~~~~
+
+.. todo:: rename to avoid confusion with validation
```

Comment:
We could rename "validity" to "consistency". The former is ARM model terminology, the later is C++ model terminology.

Reply:
Consistency sounds good to me! I went ahead and renamed it.